pre_commit/languages/docker.py

```diff
@@ -15,6 +18,32 @@
 healthy = helpers.basic_healthy
 
 
+def is_in_docker() -> bool:
+    try:
+        with open('/proc/1/cgroup') as f:
+            return 'docker' in f.read()
+    except FileNotFoundError:
+        return False
+
+
+def get_docker_path(path: str) -> str:
+    if not is_in_docker():
+        return path
+    hostname = socket.gethostname()
+    docker_output = json.loads(
+        subprocess.check_output(['docker', 'inspect', hostname]),
```

Comment:
this should use `cmd_output_b` which normalizes a bunch of stuff

Reply:
done